packages/filesystem/package.json

```diff
@@ -60,7 +61,8 @@
     "test:watch": "theiaext test:watch"
   },
   "devDependencies": {
-    "@theia/ext-scripts": "^1.4.0"
+    "@theia/ext-scripts": "^1.4.0",
+    "@types/yargs": "^11.1.0"
```

Comment:
This should go to `dependencies`.

Reply:
why @types should be in dependencies? aren't they required only on dev?

Reply:
By convention, we put the typings next to the dependency. See: https://github.com/eclipse-theia/theia/commit/81fb0c61ddbfb6274abd1918aad81bb79eb8c4a9

Reply:
We add types to dependencies that downstream don't need to add them as dev dependencies. Yes It is a convention.